dynamic/binary_test.go

```diff
@@ -1,7 +1,9 @@
 package dynamic
 
 import (
+	"reflect"
 	"testing"
+	"unsafe"
```

Comment:
You don't need "unsafe" to do this. See my suggestion, about how to do this using only the "reflect" package.